tests/unit/image_streamer/resources/test_os_volumes.py

```diff
@@ -27,7 +27,7 @@
 class OsVolumesTest(TestCase):
     def setUp(self):
         self.host = '127.0.0.1'
-        self.connection = connection(self.host)
+        self.connection = connection(self.host, 800)
```

Comment:
800 is used across all the tests , can you put it as a global variable? It will be lot easier to change when 800 is also deprecated.